tests/integration/api/test_activation_with_credential.py

```diff
@@ -347,6 +347,75 @@ def test_is_activation_valid_with_run_job_template_and_no_token_no_credential(
     ) in message
 
 
+@pytest.mark.django_db
+def test_is_activation_valid_with_updated_credential(
+    preseed_credential_types,
+):
+    fks = create_activation_related_data(TEST_EXTRA_VAR)
```

Comment:
We should replace this data creation with fixtures from conftest.py

Reply:
yeah, updated.

Reply:
@hsong-rh re-opening this since it is not updated.

Reply:
Updated to use fixture